pkg/expression/builtin_cast.go

```diff
@@ -1912,6 +2086,28 @@ func (b *builtinCastJSONAsStringSig) evalString(ctx EvalContext, row chunk.Row)
 	return s, false, nil
 }
 
+type builtinCastVectorFloat32AsStringSig struct {
+	baseBuiltinFunc
+}
+
+func (b *builtinCastVectorFloat32AsStringSig) Clone() builtinFunc {
+	newSig := &builtinCastVectorFloat32AsStringSig{}
+	newSig.cloneFrom(&b.baseBuiltinFunc)
+	return newSig
+}
+
+func (b *builtinCastVectorFloat32AsStringSig) evalString(ctx EvalContext, row chunk.Row) (res string, isNull bool, err error) {
```

Comment:
Do we need to support vecEvalXXX for vector type?

Reply:
Yes. Currently it is not implemented though. @EricZequan will find some time to provide a vectorized version for them.

Reply:
@XuHuaiyu Vectorized version has been merged in cse. I will later merge the changes in this branch.